packages/venia-ui/__mocks__/informed.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+const informed = jest.requireActual('informed');
+const {
+    Text,
+    Input,
+    Radio,
+    TextArea,
+    Select,
+    Option,
+    Checkbox,
+    RadioGroup,
+    BasicText,
+    BasicRadio,
+    BasicRadioGroup,
+    BasicTextArea,
+    BasicSelect,
+    BasicCheckbox,
+    Scope,
+    ArrayField,
+    asField,
+    ...rest
+} = informed;
+
+const wrapWithUuidPropCheck = Component => props => {
```

Comment:
_Note to my self_

Write js docs explaining the reasons behind this module mock.

Reply:
Gonna?

Reply:
Done c65b3d3